db/migrate/20200429122446_drop_mail_methods.rb

```diff
@@ -0,0 +1,16 @@
+class DropMailMethods < ActiveRecord::Migration
+  def up
+    drop_table :spree_mail_methods
+
+    # delete mail_method preferences associated with the old MailMethod model
+    execute "DELETE FROM spree_preferences WHERE key LIKE 'spree/mail_method%'"
```

Comment:
this will skip smtp_username and smtp_password though. Is that what we want?

Reply:
yes, I think smtp_username and smtp_password are the values set in MailConfiguration and used in MailSettings.

Reply:
ah ok, got it. These are old Spree preference and not ours. I wasn't sure what you were aiming at.